### PR TITLE
P2P: Remove dead code

### PR DESCRIPTION
**Pull request by heifner**
Remove handling of unused `request_message` `normal` mode.

Related to #612

### CODE REVIEW

File: plugins/net_plugin/net_plugin.cpp [hunk 1]
```diff
@@ -3694,11 +3674,9 @@ namespace eosio {
          blk_send_branch( msg.req_blocks.ids.empty() ? block_id_type() : msg.req_blocks.ids.back() );
          break;
       case normal :
-         peer_dlog( this, "received request_message:normal" );
-         if( !msg.req_blocks.ids.empty() ) {
-            blk_send( msg.req_blocks.ids.back() );
-         }
-         break;
+         peer_wlog( this, "Invalid request_message, req_blocks.mode = normal" );
```
Comment (linh2931) [inline plugins/net_plugin/net_plugin.cpp:3677]: Is `normal` not supported any more?

Comment (heifner) [reply plugins/net_plugin/net_plugin.cpp:3677]: Has not been for a very long time. I checked v2.0.0 of eosio/eos and it was not used even in that version. It was used in v1.0.0. I didn't track it down any further.

Comment (heifner) [reply plugins/net_plugin/net_plugin.cpp:3677]: Tests are failing, so I've missed something.

File: plugins/net_plugin/net_plugin.cpp [hunk 2]
```diff
@@ -3712,12 +3690,9 @@ namespace eosio {
          }
          // no break
```
Comment (heifner) [inline plugins/net_plugin/net_plugin.cpp:3691]: I missed this `no break`.

File: plugins/net_plugin/net_plugin.cpp (hunk 1 above)
Comment (heifner) [reply plugins/net_plugin/net_plugin.cpp:3677]: Fixed

**Review by greg7mdp — APPROVED** [review]

**Review by linh2931 — APPROVED** [review]

**Comment by ericpassmore** [comment]
Note:start
category: Chores
component: P2P
summary: Cleanout dead code.
Note:end

**Pull request merged**